lib/input.js

```diff
@@ -78,7 +78,8 @@ export default class Input {
       };
     }
     if (this.isSpend()) {
-      const input = this.prevout;
+      const input = { ...this.prevout }; // toJSON shouldn't mutate existing objects
+      input.hash = `0x${input.hash.toString('hex')}`;
```

Comment:
`bufferToHex` from `ethereumjs-util` can be used for this operation. Like this: `bufferToHex(input.hash)`

https://github.com/ethereumjs/ethereumjs-util/blob/master/docs/index.md#buffertohex